Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -491,7 +502,8 @@ static async Task<bool> PlatformAddAttendeeToEvent(CalendarEventAttendee newAtte
             attendeeValues.Put(CalendarContract.Attendees.InterfaceConsts.AttendeeType, (int)newAttendee.Type);
 
             var resultUri = Platform.AppContext.ApplicationContext.ContentResolver.Insert(attendeeUri, attendeeValues);
-            var result = Convert.ToInt32(resultUri.LastPathSegment);
+
+            int.TryParse(resultUri.LastPathSegment, out var result);
```

Comment:
We should null check the LasPathSegment
If tryparse fails we should just return false.

Reply:
Altered accordingly